app/services/dqt_api.rb

```diff
@@ -27,7 +27,7 @@ def self.find_trn!(trn_request)
 
     teacher_account = results.first
 
-    DqtApi.unlock_teacher!(teacher_account)
+    UnlockTeacherAccountJob.set(wait: 5.minutes).perform_later(teacher_account)
```

Comment:
This shouldn't run after 5 minutes on the initial run, you can run it immediately after, could you open a follow-up PR?